test/fixture/long-running.js

```diff
@@ -2,6 +2,7 @@ import test from '../../';
 
 test.cb('slow', t => {
 	setTimeout(t.end, 5000);
+	t.pass();
```

Comment:
We shouldn't force users to use `t.pass` in this scenario. You can pass `t.end` as an error first callback, so passing `t.end` really should count as performing an assertion.